packages/components/src/Text/Text.tsx

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import { Box, system } from '@marigold/system';
 
 // Define custom properties for your component
-type TextProps = {};
+type TextProps = {
+  themeSection?: string;
+};
 
 export const Text = system<TextProps, 'span'>(
-  ({ as = 'span', variant = '', children, ...props }) => {
+  ({ as = 'span', themeSection = 'text', variant, children, ...props }) => {
```

Comment:
(1) you actually can just drop `variant` here, because you pass it via `...props` anyway.
(2) not sure if allowing `themeSection` to be changed here. This way `<Label>` just becomes `<Box>` with the `as` default changed. Maybe we shouldn't reuse Text everywhere?

Reply:
hm but your words: "And also a label IS a text component" (in issue discussion) - that's why I made it like this 😊 make sense but true it's nearly the box

Reply:
Maybe we add later more TextProps. Then It's more different to the Box.

Reply:
I know, but the implementation does not make sense. The quote still stands. Question is how we implement it. I guess we'll figure it out later (e.g. we could introduce a useText hook or create a typography helper for the theme).

We definitely should not break our agreement about what props will be exposed.

Reply:
Ok then I use the Box for the Label component.